docs/fsharp/style-guide/formatting.md

```diff
@@ -1288,28 +1396,39 @@ let d =
         printfn "%A" x
 ```
 
-The following are non-compliant:
+These are non-compliant: 
 
 ```fsharp
 // ❌ Not OK, code formatters will reformat to the above by default
 let a = """
 foobar, long string
 """
 
-type File =
-    member this.SaveAsync(path: string) : Async<unit> = async {
-        // IO operation
-        return ()
+let d = while f do
+    printfn "%A" x
+```
+
+Record type instantiations may also place the brackets on their own lines:
+
+```fsharp
+// ✔️ OK
+let c =
+    { 
+        Name = "Bilbo"
+        Age = 111
+        Region = "The Shire" 
     }
+```
 
+You may also prefer to use "Stroustrup" style, with the opening `{` on the same line as the binding name:
+
+```fsharp
+// ✔️ OK
```

Comment:
Maybe a good section to point out that `Stroustrup` has certain limitations.
For example:
```fsharp
let c = {
    bilbo with 
    Name = "Bilbo"
    Age = 111
    Region = "The Shire" 
}
```
This leads to `Possible incorrect indentation: this token is offside of context started at position`.
However, in that example, if we further indent the update record fields:
```fsharp
let c = {
    bilbo with 
        Name = "Bilbo"
        Age = 111
        Region = "The Shire" 
}
```
it does become valid again.